requirements/install.pip

```diff
@@ -279,7 +281,7 @@ uritools==4.0.3
     # via urlextract
 urlextract==0.14.0
     # via -r requirements/install.in
-urllib3==1.25.11
+urllib3==1.26.19
```

Comment:
I think you should compute pip-compile on all impacted dep files.
It should have gotten triggered by the pip-tool pre-commit hook.
I've updated its version in https://github.com/opendatateam/udata/pull/3086/commits/5982e2a8f5ee0f24ac779887870f2415527bd9cc.